include/aspect/compat.h

```diff
@@ -22,6 +22,209 @@
 #define __aspect__compat_h
 
 #include <aspect/global.h>
+/*
+ * Fixed colorization of parallelepiped
+ */
+#if !DEAL_II_VERSION_GTE(8,4,0)
+#include <deal.II/grid/grid_generator.h>
+#include <deal.II/grid/grid_reordering.h>
+namespace dealii
+{
+  namespace GridGenerator
+  {
+    template<int dim>
+    void
+    parallelepiped (Triangulation<dim>  &tria,
```

Comment:
I personally only need the subdivided version where the subdivisions are given by a vector. Remove the other ones?